admin_guide/managing_networking.adoc

```diff
@@ -872,6 +879,81 @@ is complicated, because nodes have both "external" IP addresses and
 pass through this chain, but other pod-to-node/master traffic may
 bypass it.
 
+[[enabling-fixed-ips-for-external-project-traffic]]
+== Enabling Fixed IPs for External Project Traffic 
+
+As a cluster administrator, you can assign specific, fixed IP addresses to
+projects, so that traffic is externally easily recognizable. This is different
+from the default egress router, which is used to send traffic to specific
+destinations.
+
+Recognizable IP traffic increases cluster security by making the origins
+visible. Once enabled, all outgoing external connections from the specified
+project will share the same, fixed source IP, meaning that any external
+resources can recognize the traffic.
+
+Unlike the egress router, this is subject to `EgressNetworkPolicy` firewall
+rules.
+
+[IMPORTANT]
+====
+Enabling fixed IPs for external project traffic is a Technology Preview feature
+only. 
```

Comment:
not sure if it would be "in character" for our docs, but maybe add a note here about the fact that we plan to add further functionality to this feature in the future.

Reply:
I think you're right that it's not too in character. It's more worthwhile putting that information in when it's available, I think (and I think that's how we usually do it).

If peer review people disagree, please let me know.

Reply:
@bfallonf I think I have seen such references (I am not a fan, but....)
For ex:
`A future release of {product-title} will feature specific support for per-project back up and restore.`